README.md

```diff
@@ -16,45 +28,75 @@ video:
-1. Select a process in the list of currently running processes in the connection setup dialog, and click <b><samp>Start Session</samp></b>.
-2. The list of loaded modules will appear on the top of the <b><samp>Symbols</samp></b> tab.  If debug symbols were found for a module, it will be highlighted in blue.
-3. <b><samp>Right click</samp></b> on the module(s) for which you want to load debug information and select <b><samp>Load Symbols</samp></b>. The <b><samp>Functions</samp></b> tab will get populated.
-4. Select functions you wish to profile in the <b><samp>Functions</samp></b> tab by <b><samp>right clicking</samp></b> and choosing <b><samp>Hook</samp></b>.
-5. In the <b><samp>Capture</samp></b> tab, start profiling by pressing <kbd>F5</kbd>.  To stop profiling, press <kbd>F5</kbd> again. You can either zoom time using <kbd>W</kbd> and <kbd>S</kbd> or <kbd>ctrl</kbd> + the scroll wheel.  You can also <kbd>ctrl</kbd>+<b><samp>right-click</samp></b> and drag to zoom to a specific time range. To scale the UI, press <kbd>ctrl</kbd> + <kbd>+</kbd>/<kbd>-</kbd>. Press <kbd>SPACE</kbd> to see the last 2 seconds of capture.
-6. You can select sections of the per-thread sampling event track to get a sampling report of your selection.
+1. Select a process in the list of currently running processes in the connection
+   setup dialog, and click <b><samp>Start Session</samp></b>.
+2. The list of loaded modules will appear on the top of the
+   <b><samp>Symbols</samp></b> tab. If debug symbols were found for a module,
+   it will be highlighted in green.
+3. <b><samp>Right click</samp></b> on the module(s) for which you want to load
+   debug information and select <b><samp>Load Symbols</samp></b>. The
+   <b><samp>Functions</samp></b> tab will get populated.
+4. Select functions you wish to profile in the <b><samp>Functions</samp></b> tab
+   by <kbd>Right-Click</kbd> and choosing <b><samp>Hook</samp></b>.
+5. In the <b><samp>Capture</samp></b> tab, start profiling by pressing
+   <kbd>F5</kbd>. To stop profiling, press <kbd>F5</kbd> again. You can either
+   zoom time using <kbd>W</kbd> and <kbd>S</kbd> or <kbd>ctrl</kbd> + the scroll
+   wheel. You can also <kbd>ctrl</kbd>+<b><samp>right-click</samp></b> and drag
+   to zoom to a specific time range. To scale the UI, press <kbd>ctrl</kbd> +
+   <kbd>+</kbd>/<kbd>-</kbd>. Press <kbd>SPACE</kbd> to see the last 2 seconds
+   of capture.
+6. You can select sections of the per-thread sampling event track to get a
+   sampling report of your selection.
 
 ## Presets
 
-Once you have loaded the debug information for your modules and have chosen functions of interest, you can save your profiling preset so that you won't have to do this manually again.  To save a preset, go to <b><samp>File</samp></b> ⇒ <b><samp>Save Preset</samp></b>
+Once you have loaded the debug information for your modules and have chosen
+functions of interest, you can save your profiling preset so that you won't
```

Comment:
Again, I would specify that this is about dynamic instrumentation.

Reply:
Done.